package.json

```diff
@@ -30,8 +30,8 @@
   ],
   "dependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@edx/frontend-platform": "4.6.0",
-    "@edx/paragon": "^20.44.0",
+    "@edx/frontend-platform": "6.2.0",
+    "@edx/paragon": "21.5.6",
```

Comment:
Paragon has moved to the `openedx` scope (`@openedx/paragon`).  Version `21.5.7` is the same content as `21.5.6` of `@edx/paragon`

To do this bump, you'll need to update to the new scope and update any imports of paragon to point to the new location.

Reply:
There seems to be a [PR](https://github.com/openedx/react-unit-test-utils/pull/21) already open for it, once merged I will rebase